frontend/src/metabase/plugins/index.ts

```diff
@@ -74,7 +78,7 @@ export const PLUGIN_AUTH_PROVIDERS = [] as any;
 
 // Only show the password tab in account settings if these functions all return true.
 // Otherwise, the user is logged in via SSO and should hide first name, last name, and email field in profile settings metabase#23298.
-export const PLUGIN_IS_PASSWORD_USER: ((user: User) => boolean)[] = [];
+export const PLUGIN_IS_PASSWORD_USER: ((user: User | null) => boolean)[] = [];
```

Comment:
Can we still keep the signature as is? If you look at some of the usage
https://github.com/metabase/metabase/blob/c622038e34947dedbb6978b87b6ebab346454825/frontend/src/metabase/plugins/builtin/auth/jwt.js#L3

It's clear that that function doesn't expect a `user` to be nullable. If this causes error somewhere else you could cast the argument to be `User` in the same spirit as other files in this PR.

https://github.com/metabase/metabase/pull/26477/files#diff-4aeb510eee744ac8cce724e9c015559b0fade8862f9812ef084ef7bc0688d61aR10